apps/comps/components/trophy-badge/index.tsx

```diff
@@ -22,22 +22,56 @@ interface TrophyBadgeProps {
   size?: number;
 }
 
+// =================================================================
+// ONE PLACE TO SCALE & CONFIGURE
```

Comment:
Minor nit: this is just a style nit!!!! I'm  not a big fan of these comments. Also not sure why this has to be  in a config object. These can just be standalone constants.